synonymdev/bitkit-react-native
# feat: enable RBF, improve CPFP

### Description

- RBF now work
- e2e tests for boosing CPFP and RBF
- RBF is disabled by default, can be enabled in DEV settings. Walllet needs to be restarted 
- clonin data when it is saved to redux from beignet to avoid object feeze error

### Linked Issues/Tasks

closes #1645 #2113 

### Type of change

New feature

### Tests

Detox test

### QA Notes

- enable RBF in dev settings
- restart the app
- create send tx, make sure it is RBF
- try to boost the fee, should be RBF
